testers/create_test_env.sh

```diff
@@ -41,6 +41,7 @@ create_venv() {
     fi
     echo "${TESTERDIR}/server" > ${VENVDIR}/lib/python${PYVERSION}/site-packages/markus_${TESTERNAME}.pth
     echo "${THISSCRIPTDIR}" > ${VENVDIR}/lib/python${PYVERSION}/site-packages/markus.pth
+    echo "${TESTERDIR}/server/lib" > ${VENVDIR}/lib/python${PYVERSION}/site-packages/markus_lib.pth
```

Comment:
Is there a reason we're not creating a single .pth file and appending to it?

Otherwise yes I agree it probably should go in the customize_test_env.sh but then we'd have to pass it ${TESTERDIR}, ${VENVDIR} and ${PYVERSION} variables which starts to get messy so I thought it would be cleaner since it's just one line. 

Also, it is possible that we will introduce tester specific lib/ directories for other testers, and I don't believe there are any weird side effects of having a .pth file that points to a non-existant directory.

Reply:
We should really do a single one with multiple lines, you're right.
Leave it there, maybe behind a check for the existence of that dir. Don't know if errors are raised in case it's missing.

Reply:
I don't think there is but I'll add check just to be safe